### PR TITLE
Improve parsing of reopen_duration

### PR DESCRIPTION
**Pull request by tdabasinskas**
Hey,

I was looking into the code when I've noticed the functions related to `Duration` parsing. I might be missing something, but is there a reason all these functions can be removed, simply relying on built-in parsing of `time.Duration`? 

I've done some testing, and after removing all these functions everything still seems to be working. The only difference is that `/config` endpoint now shows `reopen_duration` full form (`3h0m0s` instead of `3h`), but I don't think that's an issue.

Again, please let me know if I've missed some reasoning behind having that custom logic 🙂 

Thanks!

### CODE REVIEW

**Comment by free** [comment]
Hi Tomas,

And thanks for contributing!

The reason behind the custom duration parsing code was to make it fully compatible with [Prometheus range durations](https://prometheus.io/docs/prometheus/latest/querying/basics/#range-vector-selectors), which are simply a number followed by `s`/`m`/`h`/`d`/`w`/`y`. It was even [lifted wholesale](https://github.com/prometheus/common/blob/7e9e6cabbd393fc208072eedef99188d0ce788b6/model/time.go#L180) from Prometheus' time handling code.

Prometheus durations are somewhat different from Go's Duration, in that they cover the second - year range, as opposed to microsecond to hour. And Prometheus doesn't accept (or output) more than one time unit in a duration (like the `3h0m0s` you mention; that would simply be `3h` in Prometheus world).

To summarize, this was done in order to keep the JIRAlert configuration as close as possible to the Prometheus configuration and thus reduce the cognitive load. (o:

So if you don't mind, I'd like to keep it as is, even though it is reinventing the wheel and ultimately may diverge from Prometheus over time.

**Comment by tdabasinskas** [comment]
Hi @free,

Oh, OKey, I wasn't ware about that. I agree it makes sense to keep the configuration as close as possible to Prometheus.

Thanks!
